daliuge-engine/dlg/apps/dockerapp.py

```diff
@@ -554,15 +555,17 @@ def run(self):
 
             # Wrap everything inside bash
             if len(cmd) > 0 and not self._noBash:
-                cmd = (
+                self._entryPoint = (
                     '/bin/bash -c "%s"'
                     % (utils.escapeQuotes(cmd, singleQuotes=False)).strip()
                 )
                 logger.info("Command after user creation and wrapping is: %s", cmd)
             else:
-                cmd = f"{utils.escapeQuotes(cmd, singleQuotes=False)}".strip()
+                # cmd = f"{utils.escapeQuotes(cmd, singleQuotes=False)}".strip()
+                # cmd = cmd.split(" ")
                 logger.info(
-                    "executing container with default cmd and wrapped arguments: %s",
+                    "executing container with default cmd %s and wrapped arguments: %s",
+                    self.defaultEntryPoint,
```

Comment:
I can't see anywhere in the code that we assign `self._entryPoint = self.defaultEntryPoint`. 

Does this mean the defaultEntryPoint is stored in the docker image, and we don't need to worry about it? If so, we currently don't do any error checking in the case where we have `len(cmd) == 0` and also no DefaultEntryPoint configured for the docker app. 

If we don't have a default entry point, it might be worth creating a logging warning for the user to help debugging in case the docker image is not given any entry points (e.g. "Running DockerApp without default entrypoints).